identity-enabler/deviceId-mobile-app/src/pages/CreatePresentation.svelte

```diff
@@ -37,32 +33,19 @@
         loadingScreen.set();
     });
 
-    function goBack() {
-        if ($modalStatus.active) {
-            modalStatus.set({ active: false, type: null });
-            return;
-        }
-
-        window.history.back();
-    }
-
     function onClickDev() {
         navigate("/tutorial");
     }
 
     function showJSON() {
-        modalStatus.set({
-            active: true,
-            type: "code",
-            props: { code: JSON.stringify(vp, null, 2) }
-        });
+        navigate("/code", { state: { code: JSON.stringify(vp, null, 2) } });
```

Comment:
I don't know if navigating to a different page is a good idea. Removing the modal now leads to a very bad performance when going back to the data matrix page ....

Reply:
Performance issues will get addressed in future PR https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/76